### PR TITLE
Update from study-template

### PR DESCRIPTION
**Pull request by bernardlee**
Refresh for cohort 016.

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -32,24 +34,32 @@ array.
...
 If I modify a class in Ruby will already created instances of that class have
 the methods I added?
 
 ```ruby
   # your answer here
 ```
 
+## Iteration
+
 Please give an example of iterating in Ruby?
```
Comment (BenGitsCode) [inline study.md]: That's not a question.

File: study.md [hunk 2]
```diff
@@ -32,24 +34,32 @@ array.
...
+## Instantiation
 How do you create a class in Ruby?
 
 ```ruby
   # your answer here
 ```
 
+## Class Modification
+
 If I modify a class in Ruby will already created instances of that class have
```
Comment (BenGitsCode) [inline study.md]: reflow 

`If I modify a class in Ruby, will previously created instances of that class have the methods I added?`

something like this maybe?

**Review by BenGitsCode — COMMENTED** [review]
Was this not updated from study template? The comment about the `?` that should be a `.` is trivial but I would still change it.

**Comment by bernardlee** [comment]
I did use study-template to update this.  I've rewritten the questions and pushed up the changes just now.

**Review by BenGitsCode — COMMENTED** [review]
Good edits in 06a6f91c 👍

**Pull request merged**